README.md

```diff
@@ -93,6 +93,19 @@ parts of a single large `DataFrame`.
 
 You can also combine any of the above options at the same time.
 
+Reading in chunks also supports progress bars
+```python
+from progressbar import ProgressBar
+pbar = ProgressBar()
+for df in pbar(read_root('bigfile.root', chunksize=100000)):
+    # process df here
+
+# or
+from tqdm import tqdm
+for df in tqdm(read_root('bigfile.root', chunksize=100000), unit="chunks"):
```

Comment:
Just changing this to be consistent (and trying out the GitHub suggestion feature).

```suggestion
for df in tqdm(read_root('bigfile.root', chunksize=100000), unit='chunks'):
```

Reply:
As I don't have permission to push to this branch I'll merge and apply the suggestion manually.

Thanks for the PR 👍

Reply:
oh, i just clicked apply to see if it works